assets/prototype/domain/shared/predicates/priceTypes/selectionPredicates/index.test.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * ?Internal dependencies
+ */
+import { PriceBasetype } from '../../../../../domain/eventEditor/data/types';
+import { getDefaultPriceType } from './index';
+import { nodes as priceTypes } from '../../../../../domain/eventEditor/data/queries/priceTypes/test/data';
+
+describe('getDefaultPriceType', () => {
+	it('should get default price type - SURCHARGE', () => {
+		const defaultPriceType = getDefaultPriceType(priceTypes);
+		expect(defaultPriceType.baseType).toBe(PriceBasetype.SURCHARGE);
```

Comment:
`defaultPriceType` doesn't always need to be a `SURCHARGE`

Reply:
Am I right @tn3rb ?

Reply:
Since `getDefaultPriceType` returns `find(isFlatFeeSurcharge)(priceTypes)` I was expecting `SURCHARGE` :smiley:

Reply:
Well first off, a better name for "default price type" would have been something like "default price modifier", since that is basically the default option for the price type selector in the ticket price calculator when you go to add a new price modifier.

Currently we will be using surcharge for the default price type(modifier) so what Vadim has is correct.
We could decide at some point in the future to change that, but if that happens we'll need to change the tests too.

What would be ideal is to have an option where the user could decide what they want to use for default price modifiers, then set that value in the config, and then read that value whenever we have to add a new price modifier. 

But what is here will be fine for the foreseeable future.